src/Orleans.SyncWork/SyncWorker.cs

```diff
@@ -44,11 +44,11 @@ public Task<bool> Start(TRequest request)
     {
         if (_task != null)
         {
-            _logger.LogDebug("{nameof(Start)}: Task already initialized upon call.", nameof(Start));
+            _logger.LogDebug($"{nameof(Start)}: Task already initialized upon call.", nameof(Start));

```

Comment:
Hey there - i'm good with the idea of fixing this up, cuz they were wrong 😓 .  But these are structured logging messages, so they shouldn't be interpolated with `$`.  Just using this as an example:

Was:

```cs
_logger.LogDebug("{nameof(Start)}: Task already initialized upon call.", nameof(Start));
```

Your change:

```cs
_logger.LogDebug($"{nameof(Start)}: Task already initialized upon call.", nameof(Start));
```

But should actually (seemingly) be:

```cs
_logger.LogDebug("{Start}: Task already initialized upon call.", nameof(Start));
```

The same idea looks to apply to each one of the changes

Reply:
Very good point. I must have been blind to the parameters passed to the log messages :) 
 I changed this now to make more semantic sense.